rules/build.ts

```diff
@@ -26,7 +26,7 @@ export async function buildAutoconsentRules(): Promise<AutoConsentCMPRule[]> {
     const normalRules = await Promise.all(files.map((file) => readFileJSON(path.join(autoconsentDir, file))));
     const generatedRules = await Promise.all(generatedFiles.map((file) => readFileJSON(path.join(generatedRulesDir, file))));
 
-    return [...normalRules, ...generatedRules];
+    return [...normalRules, ...deduplicateRules(generatedRules)];
```

Comment:
Why not apply deduplication to all rules? I know it probably won't actually do much, but it's easier to think of deduplication as a standard global build step.